### ISSUE TITLE
Generic import not working: called `Option::unwrap()` on a `None` value

### ISSUE THREAD

**Issue by Kwbmm** [IgnisDa/ryot#703]
I am attempting to import my TVTime data to Ryot.

Unfortunately, the TVTime export I have been given, is not compatible with the script at https://github.com/SirMartin/TvTimeToRyot suggested in the documentation. I believe that TVTime has since migrated their DB schema and the result is now quite different.

Anyway, I came up with a Python script to produce a JSON file according to [these](https://ignisda.github.io/ryot/guides/exporting.html#type-definition) specifications. Here is a test result:

```json
{
    "collections": [], 
    "identifier": "1705",
    "lot": "Show",
    "reviews": [],
    "source": "Tmdb",
    "source_id": "1705",
    "seen_history": [{
        "progress": 100,
        "show_episode_number": 1,
        "show_episode_series": 1,
        "ended_on": "2024-03-16T00:00:00.000Z",
        "started_on": "2024-03-16T00:00:00.000Z"
    }]
}
```

When importing this, the UI gives no feedback other than "the import is in progress" and the logs show this error:

```
ryot     | [backend] thread 'tokio-runtime-worker' panicked at apps/backend/src/importer/generic_json.rs:15:10:
ryot     | [backend] called `Option::unwrap()` on a `None` value
ryot     | [backend] note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by Kwbmm** [comment]
Also, once the import is started, import element stays in the UI and there's no way to make it go away. It disappears after some point and the arrow doesn't do anything:

![image](https://github.com/IgnisDa/ryot/assets/8115374/b9b23873-9e06-4747-885c-747be25f25c8)

**Comment by IgnisDa** [comment]
Please send me your tvtime export (email in profile). I'll add it as an official import source.

**Comment by IgnisDa** [comment]
As for the original error, it will be addressed by #685.

**Comment by Kwbmm** [comment]
> Please send me your tvtime export (email in profile). I'll add it as an official import source.

I'm still working on it. I'm waiting to have a working import on Ryot to test what's the result. I plan to put this on github
